src/coreclr/pal/src/eventprovider/lttngprovider/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ add_dependencies(coreclrtraceptprovider generated_eventing_headers)
 find_library(LTTNG NAMES lttng-ust)
 
 target_link_libraries(coreclrtraceptprovider
+                      PUBLIC
```

Comment:
@jkoritzinsky - quick Q. Why is this one public?

Reply:
I think it's public because the default to match the previous behavior is `PUBLIC`. I'll go make it PRIVATE.